CryptoPrices/Sources/MyCoin/Sources/MyCoin/ChartSectionView/PriceLineChartView.swift

```diff
@@ -22,16 +22,15 @@ struct PriceLineChartView: UIViewRepresentable {
 
     func updateUIView(_ uiView: LineChartView, context: Context) {
         let dataSet = LineChartDataSet(entries: entries)
-        // TODO: Localize this text for no data case in the integration task
-        uiView.noDataText = "No Data"
-        uiView.data = LineChartData(dataSet: dataSet)
+        uiView.data = entries.isEmpty ? nil : LineChartData(dataSet: dataSet)
         uiView.rightAxis.enabled = false
         uiView.leftAxis.enabled = false
         uiView.xAxis.enabled = false
         uiView.legend.enabled = false
         uiView.minOffset = 0.0
         uiView.isUserInteractionEnabled = false
```

Comment:
Do you think for this configuration we should put them in the `makeUIView` function because we want to set them once instead of updating along with view changes? please correct me 🙏 .

Reply:
that is definitely a valid point, I can update it in this PR too, thanks for the raise 🙏